js/script.js

```diff
@@ -56,26 +56,57 @@ function escapeHTML(text) {
 			$('#commentsTabView_close_button').on('click', function () {
 				self.commentsTabView.setObjectId(0);
 			});
-			$('#announcement_options_button').on('click', function() {
+			$('#announcement_options_button').on('click', function () {
 				$('#announcement_options').toggleClass('hidden');
 			});
 			$('#groups').each(function (index, element) {
 				self.setupGroupsSelect($(element));
 			});
 		},
 
-		_onPopState: function(params) {
+		_onPopState: function (params) {
 			params = _.extend({
 				announcement: 0
 			}, params);
 
-			console.log('_onPopState' + params.announcement);
 			this.highlightAnnouncement(params.announcement);
 		},
 
-		_onHighlightAnnouncement: function(event) {
+		_onHighlightAnnouncement: function (event) {
 			var $element = $(event.currentTarget),
-				announcementId = $element.data('announcement-id');
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 94a229ff0627a82d8cd678e1943582b9fc65d6ea
```

Comment:
There are a lot conflicts in here.

Would it be possible to make tinier patches instead of a PR with 1.2k lines changed? :smile_cat:

Reply:
Absolutely, it was a beautifier alteration I will reverse. @nickvergessen